opds.py

```diff
@@ -200,7 +200,15 @@ def categories(cls, work):
     @classmethod
     def authors(cls, work, license_pool, edition, identifier):
         """Create one or more <author> tags for the given work."""
-        return [AtomFeed.author(AtomFeed.name(edition.author or ""))]
+        authors = list()
+        for author in edition.author_contributors:
+            authors.append(AtomFeed.author(AtomFeed.name(
+                author.display_name or author.sort_name
+            )))
+
+        if authors:
+            return authors
+        return [AtomFeed.author(AtomFeed.name(""))]
```

Comment:
As long as we're doing this, it would be good to use `atom:author` tags for contributors whose role is in `Contributor.AUTHOR_ROLES`, and `atom:contributor` tags for everyone else.

I don't have a problem using `sort_name` as a fallback when we don't have `display_name`. In almost all real cases we will have `display_name`, even if it's just a guess.